packages/lesswrong/lib/reviewUtils.ts

```diff
@@ -48,9 +48,15 @@ export function postEligibleForReview (post: PostsBase) {
   return true
 }
 
+export function postIsVoteable (post: PostsBase) {
+  return getReviewPhase() === "NOMINATIONS" || post.positiveReviewVoteCount > 0
```

Comment:
Allows us to hide voting/review UI for posts that didn't get at least one positiveReviewVote during the Review Phase